sysID/fit_flywheel_arms_philip.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from models import cartpole_arm_flywheel as model
+
+# Load the data
+q = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_arms_state_q.npy')
+qDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_arms_state_qdot.npy')
+qDotDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_arms_state_qddot.npy')
+
+params = model.default_params()
+
+# convert to meters
+q[:, 0] = q[:, 0]/1000.
+q[:, 3] = q[:, 3]/1000.
+
+qDot[:, 0] = qDot[:, 0]/1000.
+qDot[:, 3] = qDot[:, 3]/1000.
+
+qDotDot[:, 0] = qDotDot[:, 0]/1000.
+qDotDot[:, 3] = qDotDot[:, 3]/1000.
+
+# convert angle to radians
+q[:, 1] = q[:, 1]*np.pi/180.
+qDot[:, 1] = qDot[:, 1]*np.pi/180.
+qDotDot[:, 1] = qDotDot[:, 1]*np.pi/180.
+
+# convert to meters squared
+q[:, 2] = q[:, 2]/1000000.
```

Comment:
I'm not sure what's going on here... what's in meters squared?

Reply:
This and all of the other comments should be fixed by the measuring rotation PR